service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/GeneratedOrderControllerMidEventTest.java

```diff
@@ -598,4 +603,37 @@ private DocumentReference expectedDraftDocument() {
             return childElement;
         }
     }
+
+    @Nested
+    class PopulateEPOTypeAndAddressMidEvent {
+
+        private final String callbackType = "populate-epo-parameters";
+
+        @Test
+        void shouldPrePopulateEpoOrderFieldsIfPresentInCaseData() {
+            AboutToStartOrSubmitCallbackResponse callbackResponse = postMidEvent(
+                buildCaseData(REMOVE_TO_ACCOMMODATION), callbackType);
+
+            CaseData caseData = extractCaseData(callbackResponse);
+
+            assertThat(caseData.getOrders().getEpoType().equals(REMOVE_TO_ACCOMMODATION));
```

Comment:
maybe this test is not needed since we don't prepopulate when REMOVE_TO_ACCOMMODATION is selected

Reply:
removed